packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -106,7 +106,10 @@ function SubTaskPopup({
         rndProps={rndProps}
         titleColor=''
       >
-        <Box gap='small'>
+        <Box 
+        className='subtaskpopup-element-that-ignores-drag-actions'
+        gap='small'
+        >
```

Comment:
```suggestion
        <Box 
          className='subtaskpopup-element-that-ignores-drag-actions'
          gap='small'
        >
```

Missing indentation here.